corehq/apps/locations/models.py

```diff
@@ -712,6 +712,13 @@ def sql_location(self):
         # For backwards compatability
         return self
 
+    @property
+    def related_location_ids(self):
+        a = LocationGroup.objects.filter(location_a=self.location_id).values_list('location_b', flat=True)
+        b = LocationGroup.objects.filter(location_b=self.location_id).values_list('location_a', flat=True)
+
+        return set(a).union(set(b))
```

Comment:
This feels kinda hacky - did you look into representing `LocationGroup` as a "through" model for a [symmetrical](https://docs.djangoproject.com/en/2.0/ref/models/fields/#django.db.models.ManyToManyField.symmetrical) `ManyToManyField`?

I believe said `ManyToManyField` would behave like `related_location_ids` here, but return a queryset rather than IDs.

Reply:
Will look into this in a separate PR

Reply:
> When defining a many-to-many relationship from a model to itself, using an intermediary model, you must use symmetrical=False (see the model field reference).

https://docs.djangoproject.com/en/1.11/topics/db/models/#extra-fields-on-many-to-many-relationships

Reply:
That seems odd, any idea why that is?  Just because you're forced to name each field on the intermediary model?